index.html

```diff
@@ -7,10 +7,42 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Portfolio Site</title>
     <link rel="stylesheet" href="./style.css">
-    <link rel="icon" href="./favicon.ico" type="image/x-icon">
+    <link href='https://fonts.googleapis.com/css?family=Playfair Display' rel='stylesheet'>
+    <link href="https://fonts.cdnfonts.com/css/poppins" rel="stylesheet">
 </head>
 
 <body>
+    <header>
+        <nav>
+            <div class="menu">
+                <div class="shape">
+                    <div id ='triangle'></div>
+                    <div id ='diamond'></div>
+                </div>
+                <div>
+                    <img id="menuimg" src="./assets/img/menu.png" alt="square">
+                </div>
+            </div>
+        </nav>
+    </header>
+    <main>
+        <section id="intro">
+            <div>
+                <div class="title">
+                    <h1>Nsong Fitz Gerald</h1>
+                </div>
```

Comment:
- [Optional] Kindly, remove unnecessary tags you can make `class='title'` directly in `<h1>` and remove the above `<div>` tag.